website/source/docs/agent/config-entries/proxy-defaults.html.md

```diff
@@ -41,6 +41,8 @@ config {
 
 - `Name` - Must be set to `global`
 
+- `Namespace` **Enterprise Only** Specifies the namespace the config entry will apply to.
```

Comment:
The namespace was added in the example for `service-defaults`, should it be added here as well?

Reply:
Done!